books/free-programming-books-subjects.md

```diff
@@ -658,7 +658,7 @@ Books that cover a specific programming language can be found in the [BY PROGRAM
 * [Securing Wireless Networks for the Home User Guide](https://mohamedation.com/securing-wifi/en/) - Mohamed Adel (HTML)
 * [The TCP/IP Guide](http://www.tcpipguide.com/free/t_toc.htm)
 * [Understanding IP Addressing: Everything you ever wanted to know](http://pages.di.unipi.it/ricci/501302.pdf) (PDF)
-* [ZeroMQ Guide](http://zguide.zeromq.org/page%3Aall)
+* [ZeroMQ Guide](https://zguide.zeromq.org/page%3Aall)
```

Comment:
this url fails the check

Reply:
yes sorry. i went through them one by one, it must have slipped from me. sorry about that.